tests/page-objects/forms/generic-form.po.js

```diff
@@ -1,4 +1,11 @@
 const helper = require('../../helper');
+const waitForLoaderToDisappear = () => {
+  try {
+    helper.waitElementToDisappear(by.css('.loader'));
```

Comment:
We use `.loader` everywhere so this is probably not specific enough - can you provide a more specific selector before the `.loader` so it's more reliable?

Reply:
removed - it seems stable enough on travis...something slow on my machine -